web/sites/default/config/webform.webform.report_campsite.yml

```diff
@@ -767,15 +774,15 @@ handlers:
       requester_name: contact_name
       requester_email: contact_email
       subject: 'Campsite ADA Accommodation Request'
-      comment: "A campsite has been reported and the community member requested an ADA accommodation:\r\n\r\n[webform_submission:values:html]"
-      tags: 'drupal webform campsite'
+      comment: "A campsite has been reported and the community member requested an ADA accommodation:\r\n\r\n[webform_submission:values:html]\r\n\r\n<br><strong>Related campsite report:</strong>\r\n[webform_submission:values:report_ticket_id]"
```

Comment:
Adds the campsite report ticket ID to the ticket description for ADA coordinators.